.github/workflows/golang-build-test.yml

```diff
@@ -22,6 +22,16 @@ jobs:
           filters: |
             filesChanged:
               - [".github/workflows/golang-build-test.yml", "src/go/**"]
+      - name: Save should_not_skip output
+        if: always()
+        run: |
+          mkdir -p ./pr
+          echo -n ${{ !steps.changes.outputs.filesChanged }} > ./pr/skipped
```

Comment:
here too

Reply:
@quentinDERORY made changes and rebased.